tools/bazar/fields/BazarField.php

```diff
@@ -209,4 +213,20 @@ public function getSemanticPredicate()
     {
         return $this->semanticPredicate;
     }
+    
+    public function jsonSerialize()
```

Comment:
i love it!

Reply:
carrément, c'est une bonne idée d'avoir passé les BazarField en serializable :+1: